ccv/chains/evm/deployment/v1_7_0/operations/burn_mint_token_pool/burn_mint_token_pool.go

```diff
@@ -1,7 +1,57 @@
 package burn_mint_token_pool
 
 import (
+	"github.com/Masterminds/semver/v3"
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/smartcontractkit/chainlink-ccip/ccv/chains/evm/gobindings/generated/latest/burn_from_mint_token_pool"
+	"github.com/smartcontractkit/chainlink-ccip/ccv/chains/evm/gobindings/generated/latest/burn_mint_token_pool"
+	"github.com/smartcontractkit/chainlink-ccip/ccv/chains/evm/gobindings/generated/latest/burn_with_from_mint_token_pool"
+	"github.com/smartcontractkit/chainlink-ccip/chains/evm/deployment/utils/operations/contract"
+	"github.com/smartcontractkit/chainlink-deployments-framework/deployment"
 	cldf_deployment "github.com/smartcontractkit/chainlink-deployments-framework/deployment"
 )
 
-var ContractType cldf_deployment.ContractType = "BurnMintTokenPool"
+var BurnMintContractType cldf_deployment.ContractType = "BurnMintTokenPool"
+
+var BurnWithFromMintContractType cldf_deployment.ContractType = "BurnWithFromMintTokenPool"
+
+var BurnFromMintContractType cldf_deployment.ContractType = "BurnFromMintTokenPool"
+
+var Version = semver.MustParse("1.7.0")
+
+type ConstructorArgs struct {
+	Token              common.Address
+	LocalTokenDecimals uint8
+	AdvancedPoolHooks  common.Address
+	RMNProxy           common.Address
+	Router             common.Address
+}
+
+var bytecodeByTypeAndVersion = map[string]contract.Bytecode{
```

Comment:
LockRelease and BurnMint have diverged, can no longer use a unified constructor operation.